cli/azd/pkg/project/service_target_aks.go

```diff
@@ -129,6 +134,11 @@ func (t *aksTarget) Package(
 	serviceConfig *ServiceConfig,
 	packageOutput *ServicePackageResult,
 ) *async.TaskWithProgress[*ServicePackageResult, ServiceProgress] {
+	_, err := t.getTargetResource(ctx, serviceConfig)
+	if err != nil {
+		return nil
+	}
```

Comment:
Downstream callers are expecting this to be an async task.  I'd be better to return a task with an empty result to achieve what you are going for.

Reply:
Done